packages/@okta/vuepress-theme-prose/components/News.vue

```diff
@@ -20,40 +20,40 @@
             Blog
           </p>
           <h3 class="news__article-title dont-break-out">
-            How Authentication and Authorization Work for SPAs
+            {{ blogPostTitles[0] || 'How Authentication and Authorization Work for SPAs' }}
           </h3>
           <a
             class="news__article-link dont-break-out"
-            href="https://developer.okta.com/blog/2023/04/04/spa-auth-tokens"
+            :href="blogPostLinks[0] || 'https://developer.okta.com/blog/2023/04/04/spa-auth-tokens'"
           >
             Read the blog post
           </a>
         </div>
       </article>
       <article class="news__article">
         <p class="news__text dont-break-out">
-          Video
+          {{ blogPostTitles[1] ? 'Blog' : 'Video' }}
```

Comment:
We need to distinguish Video from Blog item and have a different configuration. The current approach will break when we get another Video.

Reply:
We will get only blogs in the RSS feed. The video that we see currently in dev docs was added explicitly.